content/en/docs/Installation/trustedboot.md

```diff
@@ -64,24 +59,66 @@ Substitute `$MY_ORG` for your own string, this can be anything but it help ident
+```
+
+### Exporting keys from BIOS/UEFI and using them
+
+This is useful if you want to handle mass-installations where you don't want to enroll the generated keys manually in your bios/uefi.
+
+Some hardware might require additional vendor keys, that could be e.g. used to sign additional firmware. In order to re-use the same set of keys in the machine, you can export the keys from the BIOS/UEFI and use them to generate the UKI files.
+
+Some BIOS/UEFI allows to export the keys to USB stick directly from the BIOS/UEFI setup menu (for example [HPE](https://techlibrary.hpe.com/docs/iss/proliant-gen10-uefi/GUID-26E9F167-8061-4D85-9A0A-B610D00DFA3B.html)). If you have the keys in a USB stick, you can copy in a directory and use them to generate the UKI files.
+
+```bash
+MY_ORG="Acme Corp"
+MACHINE_CERTS="$PWD/path/to/machine-certs"
+# ~$ tree $MACHINE_CERTS
+# /path/to/machine-certs/
+# ├── db
+# ├── dbx
+# ├── KEK
+# └── PK
+
+# Generate the keys
+docker run -v $MACHINE_CERTS:/work/machine-keys -v $PWD/keys:/work/keys -ti --rm quay.io/kairos/osbuilder-tools:latest genkey "$MY_ORG" --custom-cert-dir /work/machine-keys --expiration-in-days 365 -o /work/keys
+```
+
+{{% alert title="Warning" %}}
+
+This command can be combined with the `--skip-microsoft-certs-I-KNOW-WHAT-IM-DOING` flag to avoid auto-enrolling the Microsoft keys if not needed (or already present in the "custom certs")
+
+{{% /alert %}}
+
 ## Building installable medium
 
 To build the installable medium you need to run the following commands:
 
 {{< tabpane text=true  >}}
 {{% tab header="From a container image" %}}
 ```bash
-CONTAINER_IMAGE=quay.io/kairos/fedora:38-core-amd64-generic-v3.0.0-alpha1
-# ubuntu:
-# CONTAINER_IMAGE=quay.io/kairos/ubuntu:23.10-core-amd64-generic-v3.0.0-alpha1
-docker run -ti --rm -v $PWD/build:/result -v $PWD/keys/:/keys enki build-uki $CONTAINER_IMAGE -t iso -d /result/ -k /keys
+CONTAINER_IMAGE=quay.io/kairos/fedora:38-core-amd64-generic-{{< kairosVersion>}}-uki
+docker run -ti --rm -v $PWD/build:/result -v $PWD/keys/:/keys quay.io/kairos/osbuilder-tools:latest build-uki $CONTAINER_IMAGE -t iso -d /result/ -k /keys
+# to build an EFI file only
+docker run -ti --rm -v $PWD/build:/result -v $PWD/keys/:/keys quay.io/kairos/osbuilder-tools:latest build-uki $CONTAINER_IMAGE -t uki -d /result/ -k /keys
 ```
 {{% /tab %}}
 {{% tab header="From a directory" %}}
 ```bash
 # Assuming you have a "rootfs" directory with the content of the OS
 # If the image is in a directory ($PWD/rootfs) you can use the following command
-docker run -ti --rm -v $PWD/build:/result -v $PWD/rootfs:/rootfs -v $PWD/keys/:/keys enki build-uki dir:/rootfs/ -t iso -d /result/ -k /keys
+docker run -ti --rm -v $PWD/build:/result -v $PWD/rootfs:/rootfs -v $PWD/keys/:/keys quay.io/kairos/osbuilder-tools:latest build-uki dir:/rootfs/ -t iso -d /result/ -k /keys
```

Comment:
Should we also document this flag here? https://github.com/kairos-io/enki/pull/87/files

Reply:
already documented here: https://github.com/kairos-io/kairos-docs/pull/168/files#diff-768757df5b435417f67d573865c7a3fd5a421a866a8dbf30af981778262c50b6R374

Reply:
or did I miss anything @jimmykarily ?

Reply:
nope, I just missed it. All good.